ReactWindows/ReactNative.Net46/Bridge/DispatcherHelpers.cs

```diff
@@ -30,13 +30,21 @@ internal static Dispatcher CurrentDispatcher
                     throw new ArgumentException("Dispatcher must be an STA thread");
                 }
 
-                _dispatcher = value;
+                s_mainDispatcher = value;
             }
         }
 
+        /// <summary>
+        /// Sets the main dispatcher.
+        /// </summary>
+        public static void Initialize()
+        {
+            s_mainDispatcher = Dispatcher.CurrentDispatcher;
```

Comment:
I think we were here once before trying to use Dispatcher.CurrentDispatcher and we ran into problems if we called it on a thread that did not have a dispatcher thread. If the OnResume is guaranteed to be executed on thread with a dispatcher (e.g. the UI Thread) then this will be OK and the "MainDispatcher" will be the dispatcher on the thread that called OnResume.

If OnResume is called on a thread that does not have a dispatcher then this will likely be null and the AssertDispatcherSet will throw...